src/main/java/io/vertx/core/http/impl/Http1xClientConnection.java

```diff
@@ -87,6 +87,7 @@
   private int keepAliveTimeout;
   private int seq = 1;
   private long bytesRead;
+  private long initialTimestamp = System.currentTimeMillis();
```

Comment:
we should call this only when the option is active, now need to call it otherwise